cpp/oneapi/dal/io/detail/load_graph.hpp

```diff
@@ -60,48 +60,62 @@ void convert_to_csr_impl(const edge_list<vertex_type<Graph>> &edges, Graph &g) {
     }
 
     using vertex_t = typename Graph::vertex_type;
+    using vertex_size_type = typename Graph::vertex_size_type;
```

Comment:
using **vertex_size_t** = typename Graph::vertex_size_type;
It is good to name consistently. But it can be confused with size_t type. It is up to you.

Reply:
I do not understand  why do we use vertex_t naming. It is wrong name due to the oneAPI coding style (no Hungr. aka _t) . That is we need to provide full name vertex_type, vertex_size_type